invenio_app_ils/circulation/loaders/schemas/json/loan_request.py

```diff
@@ -104,4 +105,12 @@ def postload_checks(self, data, **kwargs):
                     "request_expire_date": [message],
                 }
             )
+        elif end - start < loan_request_offset:
```

Comment:
Was there any unit/functional test on this? Would it be possible to add tests?

In particular, I would test:
- if I put a negative value: it looks like you can do that, we should block it
- if I put `0`
- if I put a few days
- what happens if I set the config to `2 days` and I make the request now at 11am: will the start date be in 2 days, at 11am (exactly 48 hours), or the check will only compare days and I need to select the third day?

Reply:
I will look into the unit test addition for this.

Regarding the 4th point: It will disable today and tomorrow only. The check only compares on the days.